mulighetsrommet-api/src/main/kotlin/no/nav/mulighetsrommet/api/domain/dbo/ArenaMigreringTiltaksgjennomforingDbo.kt

```diff
@@ -0,0 +1,19 @@
+package no.nav.mulighetsrommet.api.domain.dbo
+
+import no.nav.mulighetsrommet.domain.dbo.Avslutningsstatus
+import no.nav.mulighetsrommet.domain.dbo.TiltaksgjennomforingTilgjengelighetsstatus
+import java.time.LocalDate
+import java.util.*
+
+data class ArenaMigreringTiltaksgjennomforingDbo(
+    val id: UUID,
+    val tiltakskode: String,
+    val navn: String,
+    val arrangorOrganisasjonsnummer: String,
+    val startDato: LocalDate,
+    val sluttDato: LocalDate?,
+    val avslutningsstatus: Avslutningsstatus,
+    val tilgjengelighet: TiltaksgjennomforingTilgjengelighetsstatus,
```

Comment:
Denne er jeg litt usikker på om vi trenger, siden dette er noe vi ikke har fått fra Arena fra før av. Dvs. jeg tror ikke Arena bryr seg om verdien vi vil legge i denne variabelen.

Reply:
Den brukes til å konvertere tilbake til en sånn JaNeiStatus som vi får fra Arena, men litt usikker om den trengs ja

Reply:
Ja, så det. Da kan den nok bare være. Poenget var uansett at jeg tror ikke Arena trenger den tilsendt, men om den brukes for å fortelle noe om JaNeiStatus så er det sikkert greit å ha den.